src/main/kotlin/no/nav/henvendelsesarkiv/abac/AbacCache.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.henvendelsesarkiv.abac
+
+import com.google.gson.GsonBuilder
+import com.google.gson.JsonObject
+import java.util.*
+
+private data class AbacCackeKey(val sub: String, val method: String, val action: String)
+
+private data class AbacCacheValue(val timestamp: Long, val hasAccess: Boolean)
+
+private val gson = GsonBuilder().create()
+
+private const val EXPIRY = 1000 * 60 * 5
```

Comment:
Er dette 5 minutter? Kanskje navngi den for å tydeliggjøre? FiveMinutes eller no slikt?